nevergrad/benchmark/optgroups.py

```diff
@@ -41,6 +41,11 @@ def get_optimizers(*names: str, seed: tp.Optional[int] = None) -> tp.List[Optim]
     return optims
 
 
+@registry.register
+def ngopts() -> tp.Sequence[Optim]:
+    return [o for o in optimizerlib_registry if "NGOpt" in o] + ["Shiwa"]
```

Comment:
Better check for all subclasses of NGOptBase
something like `[name for name, cls in optimizerlib_registry.items() if issubclass(cls, NGOptBase)]`
Relying on name is not robust on the long term